keyboards/knobgoblin/knobgoblin.c

```diff
@@ -15,3 +15,81 @@
  */
 
 #include "knobgoblin.h"
+
+#ifdef ENCODER_ENABLE
+/* assign keycodes to the encoder rotation */
+__attribute__((weak)) void encoder_update_user(uint8_t index, bool clockwise) {
+
+			if (index == 1) { /* Bottom encoder */
+				if (clockwise) {
+					tap_code(KC_VOLU);
+				} else {
+					tap_code(KC_VOLD);
+				}
+			}
+			if (index == 0) { /* Top encoder */
+				if (clockwise) {
+					tap_code(KC_MNXT);
+				} else {
+					tap_code(KC_MPRV);
+				}
+			}
+}
+#endif
+
+#ifdef OLED_DRIVER_ENABLE
+/* rotate screen for proper orentation*/
+__attribute__((weak)) oled_rotation_t oled_init_user(oled_rotation_t rotation) { return OLED_ROTATION_270; }
+
+/* byte map for the goblin logo, knob goblin text, and level text */
+static void render_logo(void) {
```

Comment:
How about `render_goblin_logo`?
`render_logo` is frequently used by users so this could create collision.

Reply:
done. thanks for the comment